subtests/docker_cli/run_volumes/run_volumes.py

```diff
@@ -42,7 +42,12 @@
 
 
 class run_volumes(SubSubtestCaller):
-    config_section = 'docker_cli/run_volumes'
+
+    def run_once(self):
+        ptc = self.config['pretest_cmd'].strip()
+        if ptc != '' and ptc is not None:
```

Comment:
You're correct, however your reminded me there is a docker autotest "gotcha" WRT config values:  See ``dockertest/config.py:272  Though it's still wrong so I'll fix it.  Thanks.